.github/workflows/main.yml

```diff
@@ -82,6 +91,18 @@ jobs:
 
     - uses: actions/checkout@v2
 
+    - uses: ilammy/msvc-dev-cmd@v1.10.0
+      with:
+        sdk: 10.0.17763.0
+
+    - name: Install toolchain (Windows)
+      run: |
+        python -m pip install scons
+        Invoke-WebRequest -Uri "https://inkscape.org/gallery/item/18067/inkscape-0.92.5-x64.exe" -OutFile "inkscape.exe"
+        Start-Process .\inkscape.exe /S -NoNewWindow -Wait
```

Comment:
I'm intrigued by this - I presume it is necessary for our `inkscape --export-png` calls from the SConstruct to work, but I don't know enough about Windows to know why. Could you explain?

Reply:
This is just a funky way of installing Inkscape, the process it's running is just the installer, it waits for it to finish then terminates.

There are lots of different installer front-ends that have different ways of doing a non-GUI installation, so there's often some experimentation needed to figure it out. If I remember correctly, the usual method of just running it without the `Start-Process` was running it asynchronously, so when Scons went to check for Inkscape at the beginning, it was a race to see if it had installed in time. Which Inkscape usually lost.

Reply:
Ah, cool. I had wrongly imagined it was launching Inkscape in some sort of server mode so that it was sitting there waiting to receive commands...